tool/site/client-react/src/pages/loginPage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useContext, useState} from "react"
+import {Button, Label} from "nhsuk-react-components"
+import {AppContext} from "../index"
+import {axiosInstance} from "../requests/axiosInstance"
+import ButtonList from "../components/buttonList"
+import {redirect} from "../browser/navigation"
+import {isDev, isInt} from "../services/environment"
+
+const LoginPage: React.FC = () => {
+  const {baseUrl, environment} = useContext(AppContext)
+
+  const [attendedAccessSelected, setAttendedAccessSelected] = useState(false)
+
+  if (isInt(environment)) {
```

Comment:
this would stop devs without smartcards from using int - is this what we want?

Reply:
short answer: yes, we should all have smartcards. will discuss in more details